specification/maps/data-plane/Creator/preview/2022-09-01-preview/examples/dataset/Dataset_List.json

```diff
@@ -25,16 +23,14 @@
               "opening": 471,
               "areaElement": 496
             },
-            "ontology": "facility-2022-09-01-preview"
+            "ontology": "facility-2.0"
```

Comment:
Why was this changed from facility-2022-09-01-preview?  Was that a mistake?

Reply:
Sorry, Just saw these comments, Good catch. Thanks!